kleinanzeigen_bot/__init__.py

```diff
@@ -443,7 +443,7 @@ def publish_ads(self, ad_cfgs:list[tuple[str, dict[str, Any], dict[str, Any]]])
             count += 1
             LOG.info("Processing %s/%s: '%s' from [%s]...", count, len(ad_cfgs), ad_cfg["title"], ad_file)
             self.publish_ad(ad_file, ad_cfg, ad_cfg_orig)
-            pause(3000, 5000)
+            self.web_await(lambda _: self.webdriver.find_element(By.ID, "checking-done").is_displayed(), timeout = 5 * 60)            
```

Comment:
Can you please remove the trailing whitespaces, it makes the build fail. See https://github.com/Second-Hand-Friends/kleinanzeigen-bot/actions/runs/3481099979/jobs/5821750360#step:8:16 Thanks!